db/migrate/20220827115952_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :string
```

Comment:
- [ ] There is no `string` column in the schema. Kindly delete it. User table only name, photo, bio and post counter column.